Core/Systems/PersistentDataSystem/PersistentDataStore.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.IO;
+using Terraria.ID;
+using Terraria.ModLoader.IO;
+
+namespace StarlightRiver.Core.Systems.PersistentDataSystem
+{
+	internal abstract class PersistentDataStore : IOrderedLoadable
+	{
+		public float Priority => 1f;
+
+		public void Load()
+		{
+			PersistentDataStoreSystem.PutDataStore(this);
+
+			string dir = Path.Join(Main.SavePath, "StarlightRiverPersistent");
+
+			if (!Directory.Exists(dir))
+				Directory.CreateDirectory(dir);
+
+			string currentPath = Path.Join(Main.SavePath, "StarlightRiverPersistent", GetType().Name);
+			LoadFromFile(currentPath);
+		}
+
+		public void Unload()
```

Comment:
remove empty method

Reply:
This is required for interface conformance